src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/Blocks/product_preferences_stock.html.twig

```diff
@@ -51,6 +43,33 @@
             {{ form_widget(stockForm.stock_management) }}
           </div>
         </div>
+
+        <div class="form-group row">
+          {{ ps.label_with_help(('Default pack stock management'|trans), ('When selling packs of products, how do you want your stock to be calculated?'|trans({}, 'Admin.Shopparameters.Help'))) }}
+          <div class="col-sm">
+            {{ form_errors(stockForm.pack_stock_management) }}
+            {{ form_widget(stockForm.pack_stock_management) }}
+          </div>
+        </div>
+
+        <div class="form-group row">
+          {{ ps.label_with_help(('Display unavailable attributes on the product page'|trans), ('If an attribute is not available in every product combination, it will not be displayed.'|trans({}, 'Admin.Shopparameters.Help'))) }}
+          <div class="col-sm">
+            {{ form_errors(stockForm.display_unavailable_attributes) }}
+            {{ form_widget(stockForm.display_unavailable_attributes) }}
+          </div>
+        </div>
+
+        {{ form_row(stockForm.display_last_quantities) }}
+
+        <div class="form-group row">
+          {{ ps.label_with_help(('Order out-of-stock products'|trans), ('By default, the "%add_to_cart_label%" button is hidden when a product is unavailable. You can choose to have it displayed in all cases.'|trans({'%add_to_cart_label%': 'Add to cart'|trans({}, 'Shop.Theme.Actions')}, 'Admin.Shopparameters.Help'))) }}
```

Comment:
Let's stick with the current wording. We'll change it later if necessary when the toggle buttons are modified. Thanks!

```suggestion
          {{ ps.label_with_help(('Allow ordering of out-of-stock products'|trans), ('By default, the "%add_to_cart_label%" button is hidden when a product is unavailable. You can choose to have it displayed in all cases.'|trans({'%add_to_cart_label%': 'Add to cart'|trans({}, 'Shop.Theme.Actions')}, 'Admin.Shopparameters.Help'))) }}
```